src/_common/sticker/collect/collect.vue

```diff
@@ -0,0 +1,244 @@
+<template>
+	<div>
+		<div class="-reveal">
+			<div v-if="!canReveal">
+				<app-sticker-card-hidden
+					v-app-tooltip="$gettext(`Not enough progress to unlock sticker`)"
+				/>
+			</div>
+			<div v-else-if="isRevealing" class="-card-revealing-outer">
+				<app-sticker-card-hidden class="-card-revealing" />
+			</div>
+			<template v-else-if="isRevealed && purchasedSticker">
+				<div class="-card-revealed-container">
+					<app-sticker-card
+						class="-card-revealed"
+						:sticker="purchasedSticker"
+						label="+1"
+					/>
+					<div
+						v-if="purchasedSticker.rarity > 0"
+						class="-card-revealed-effect"
+						:class="{
+							'-card-revealed-effect-uncommon': purchasedSticker.rarity === 1,
+							'-card-revealed-effect-rare': purchasedSticker.rarity === 2,
+							'-card-revealed-effect-epic': purchasedSticker.rarity === 3,
+						}"
+					/>
+				</div>
+				<div v-if="showCollectControls" class="-revealed-controls anim-fade-in">
+					<app-button primary @click="onClickCollect">
+						<translate>Collect</translate>
+					</app-button>
+				</div>
+			</template>
+			<template v-else>
+				<div>
+					<app-sticker-card-hidden @click.native="onBuySticker" class="-card-hidden" />
+				</div>
+				<hr class="underbar underbar-center" />
+				<div class="-collect-flavor-text">
+					<translate>What's it gonna be? The suspense is almost too much!</translate>
+				</div>
+				<small class="text-muted -sticker-amount">
+					<translate
+						:translate-n="canBuyStickerAmount"
+						:translate-params="{ count: canBuyStickerAmount }"
+						translate-plural="You can unlock %{ count } more stickers."
```

Comment:
For some reason this is showing 10 stickers for me to collect on the "collect route" instead of the 1 that I have.